crates/rerun-cli/Cargo.toml

```diff
@@ -2,7 +2,6 @@
 name = "rerun-cli"
 authors.workspace = true
 categories = ["visualization", "computer-vision"]
-default-run = "rerun"
```

Comment:
Was this intentional?

This will break just writing `cargo run` in the terminal… but that's probably for the better anyway, since `cargo rerun` does the same thing, only a lot faster